pair-filter/pair_filter/image_chooser.py

```diff
@@ -134,8 +134,22 @@ def get_image_tag(self) -> Optional[str]:
         # Does not match 'travis-ci-garnet-trusty-1503972833'
         match_obj_tag = re.search(r'(travis-ci(-[a-z]+)+-[0-9]+-[0-9a-z]+)', instance)
         if not match_obj_tag:
+            # Matches 'travisci/ci-garnet:packer-1503972846'
```

Comment:
@octaviocorona Before you make the chances I have suggested below. This isn't a tag that can be found via commit sha, right? As `1503972846` is the time stamp. Isn't this care solved by `ExactImageChooserByTag`?

Reply:
Wow, you're right. I will make appropriate changes as necessary.